apps/desktop-wallet/src/components/Tooltip.tsx

```diff
@@ -26,5 +26,5 @@ export type HasTooltip<T> = T & { tooltip?: string }
 export default styled(Tooltip)`
   background-color: rgb(10, 10, 10);
   color: #fff;
-  z-index: 1;
+  z-index: 3;
```

Comment:
Why was this needed? This tells me there is something wrong with the stacking context...

Reply:
Because of modal (z-index of 2), tooltips were shown below.

Reply:
I don't see any `z-index: 2` in a modal. I will have a look. I would like to not start re-introducing arbitrary z-index values and rather fix our stacking context.